src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ props }) => {
```

Comment:
The destructuring of props should be corrected. Instead of `({ props })`, it should be `({ name, age, sex, isMarried, partnerName })` to directly destructure the properties from the props object.